widgetssdk/src/main/java/com/glia/widgets/callvisualizer/CallVisualizerRepository.kt

```diff
@@ -19,20 +21,40 @@ class CallVisualizerRepository {
 
     fun init(callVisualizerCallback: CallVisualizerCallback) {
         this.callback = callVisualizerCallback
+        autoAcceptEngagementRequest()
+        showDialogOnMediaUpgradeRequest()
+        Logger.d(TAG, "CallVisualizerRepository initialized")
+    }
+
+    private fun showDialogOnMediaUpgradeRequest() {
         Dependencies.glia().callVisualizer.on(Omnibrowse.Events.ENGAGEMENT)
         { engagement: OmnibrowseEngagement ->
             Logger.d(TAG, "New Call Visualizer engagement started")
             val upgradeOfferConsumer = prepareMediaUpgradeOfferConsumer(engagement)
             engagement.media.on(Media.Events.MEDIA_UPGRADE_OFFER, upgradeOfferConsumer)
         }
-        Logger.d(TAG, "CallVisualizerRepository initialized")
+    }
+
+    private fun autoAcceptEngagementRequest() {
+        Dependencies.glia().callVisualizer.on(Omnibrowse.Events.ENGAGEMENT_REQUEST)
```

Comment:
I'd like to have `CallVisualizer` as a constructor parameter here

Reply:
Good point. Fixed 909b913c3b5428cff06a6414b554d104d1bcf1e5